tcms/xmlrpc/api/testplan.py

```diff
@@ -7,7 +7,7 @@
 from tcms.testplans.models import TestPlan
 from tcms.testcases.models import TestCase, TestCasePlan
 
-from tcms.xmlrpc.forms import EditPlanForm, NewPlanForm
+from tcms.xmlrpc.api.forms.testplan import NewPlanForm, EditPlanForm
```

Comment:
Unnecessary swap of class names at the end which produces more visual diff. Unless pylint or flake8 complains about it refrain from changing what doesn't need changing.